build/Build.cs

```diff
@@ -189,6 +189,33 @@ class Build : NukeBuild
         });
     });
 
+    Target PackMergedClientNugetForMacOs => _ => _
```

Comment:
You've happened to hit the issue on macOS, but neither the problem nor solution are macOS specific.

Other developers on linux will have the same problem, and if a windows dev is in your situation they may be able to run ILRepack, but the Azure code signing keys most likely won't be accessible and they also may want an unsigned nuget package.

Suggestion: Remove the 'macOS' and call this something more generic like `PackUnsignedClient`

Reply:
Also: This target is called `PackMergedClient` but it doesn't seem to be doing any ILMerging?

Reply:
I've updated the naming of targets to make it clearer and I've removed the MacOS specific naming to make it more generically creating "Unsigned" nuget packages 👍